dct_backend/webscraper.py

```diff
@@ -0,0 +1,111 @@
+import requests
+from bs4 import BeautifulSoup
+import dct
+
+# DCT import
+DCT = dct.DCT()
+
+# setup BeautifulSoup with URL
+req = requests.get("https://emoryatlanta.cafebonappetit.com/cafe/dobbs-common-table/")
+soup = BeautifulSoup(req.content, "html.parser")
+
+# Title of webpage
+title = soup.title
+# print(title.prettify()) #prettify() can be a useful beautifulsoup method
+title = title.get_text()
+
+def scrapeFood(mealtime, mealname, station):
```

Comment:
if these are soups, append `_soup` at the end.

Reply:
Accept a `DCT()` object as a parameter. Clear it first, then add each data information

Reply:
use snake casing

Reply:
What is snake casing?

Reply:
https://en.wikipedia.org/wiki/Snake_case